### PR TITLE
Move flow control specification to a single location.

### PR DESCRIPTION
**Pull request by huitema**
@martinthomson this should fix one of the problems outlined in issue #57

### CODE REVIEW

**Comment by saradickinson** [comment]
Thanks for this and all the other feedback - definite improvements to the document

File: draft-ietf-dprive-dnsoquic.md [hunk 1]
```diff
@@ -569,11 +554,27 @@ connections. For example:
 
 Servers and Clients manage flow control as specified in QUIC.
 
-Servers MAY use the "maximum stream ID" option of the QUIC transport to limit
+Servers MAY use the "maximum stream ID" parameter of the QUIC transport to limit
```
Comment (martinthomson) [inline draft-ietf-dprive-dnsoquic.md:557]: The "MAY" here is unnecessary (it's a restatement of QUIC requirements).

"can" is fine.

File: draft-ietf-dprive-dnsoquic.md [hunk 2]
```diff
@@ -569,11 +554,27 @@ connections. For example:
...
 the number of streams opened by the client. This mechanism will effectively
 limit the number of DNS queries that a client can send on a single DoQ
-connection.
+connection. The initial value of this parameter is specified by the transport
+parameter `initial_max_streams_bidi`. For better performance, it is RECOMMENDED
+that servers chose a sufficiently large value for this parameter.
```
Comment (martinthomson) [inline draft-ietf-dprive-dnsoquic.md:562]: This last sentence depends on a shared understanding of "sufficient" in this context, which is pretty fraught without more guidance.

File: draft-ietf-dprive-dnsoquic.md [hunk 3]
```diff
@@ -569,11 +554,27 @@ connections. For example:
...
+nodes at a given time. The initial values of per stream flow control parameters
+is defined by two transport parameters:
 
+* initial_max_stream_data_bidi_local: when set by the client, specifies the
+  amount of data that servers can send on a "response" stream without waiting
+  for a MAX_STREAM_DATA frame.
+
+* initial_max_stream_data_bidi_remote: when set by the server, specifies the
+  amount of data that clients can send on a "query" stream without waiting for
+  a MAX_STREAM_DATA frame.
```
Comment (martinthomson) [inline draft-ietf-dprive-dnsoquic.md:574]: I would only keep the single sentence on line 564 and delete the rest of this.  Maybe add a reference to the relevant section in QUIC.

File: draft-ietf-dprive-dnsoquic.md [hunk 4]
```diff
@@ -569,11 +554,27 @@ connections. For example:
...
+
+For better performance, it is RECOMMENDED that clients and servers set each of
+these two parameters to a value of 65535 or greater.
```
Comment (martinthomson) [inline draft-ietf-dprive-dnsoquic.md:577]: OK, now we are talking about something concrete (as a definition for "sufficient").  But this number appears out of nowhere.

**Review by martinthomson — COMMENTED** [review]
I see you merged it already, so I can't make suggestions, except by using words.  Thanks for moving things, but I think you should also delete a few things too.

**Comment by huitema** [comment]
I did not merge it -- Sara did. Let me invite you to the project so I can formally ask for your review.

-- Christian Huitema

**Pull request merged**